js-apps/meep-frontend/src/js/containers/idc-map.js

```diff
@@ -661,17 +662,23 @@ class IDCMap extends Component {
   // UE Marker Event Handler
   updateComputePopup(marker) {
     if (marker) {
-      var latlng = marker.getLatLng();
+      // Retrieve state 
       var table = this.getTable();
```

Comment:
Need a null check on table. Suggest code like in the UE popup like this:
```
    var table = this.getTable();
    if (marker && table && table.entries) {
```

Reply:
Thanks, new commit to reflect update